next/.site/layouts/partials/docs/inject/body.html

```diff
@@ -1,12 +1,25 @@
-<script
-  type="module"
-  src="https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate"
-></script>
-
+<script type="module" src="https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate"></script>
 <pwa-update swpath="{{ "/sw.js" | relURL }}"></pwa-update>
 
 {{- if default true .Site.Params.BookSearch }}
-{{- $searchJSFile := printf "%s.search.js" .Language.Lang }}
-{{- $searchJS := resources.Get "search.js" | resources.ExecuteAsTemplate $searchJSFile . | resources.Minify | resources.Fingerprint }}
-<script defer src="{{ $searchJS.RelPermalink }}" integrity="{{ $searchJS.Data.Integrity }}"></script>
-{{ end -}}
\ No newline at end of file
+  {{- $searchJSFile := printf "%s.search.js" .Language.Lang }}
+  {{- $searchJS := resources.Get "search.js" | resources.ExecuteAsTemplate $searchJSFile . | resources.Minify | resources.Fingerprint }}
+  <script defer src="{{ $searchJS.RelPermalink }}" integrity="{{ $searchJS.Data.Integrity }}"></script>
+{{ end -}}
+
+{{- if default false ($.Param "math-mode") -}}
+<script src="https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.js" integrity="sha384-y23I5Q6l+B6vatafAwxRu/0oK/79VlbSz7Q9aiSZUvyWYIYsd+qj+o24G5ZU2zJz" crossorigin="anonymous"></script>
```

Comment:
Wouldn't it make sense to use the local version that it is part of the theme?

Reply:
i will do that later